src/components/shared/Entity/Actions/Save.tsx

```diff
@@ -69,16 +83,19 @@ function EntityCreateSave({
 
     const waitForPublishToFinish = (logTokenVal: string) => {
         resetFormState(status);
-        console.log('wait for finish');
         const subscription = startSubscription(
             supabaseClient.from(
                 `${TABLES.PUBLICATIONS}:draft_id=eq.${draftId}`
             ),
             async (payload: any) => {
                 if (payload.logs_token === logTokenVal) {
+                    const formStatus = dryRun
```

Comment:
Is this used in more than one place? If not, why is the value being stored in a constant?

Reply:
It was for a minute and then it wasn't and was too lazy to put it back how it was.